packages/web/src/components/collection/CollectionCard.tsx

```diff
@@ -37,123 +38,99 @@ const cardSizeToCoverArtSizeMap = {
-              <Text variant='label' color='subdued'>
-                {save_count}
-              </Text>
-            </Flex>
-            {isPurchase ? (
-              <LockedStatusPill
-                variant='premium'
-                locked={!access.stream}
-                id={`${id}-condition`}
-              />
-            ) : null}
-          </>
-        )}
-      </Flex>
-    </Paper>
-  )
-}
+    const collection = useSelector((state) => getCollection(state, { id }))
+
+    const handleNavigate = useLinkClickHandler<HTMLDivElement>(
+      collection?.permalink ?? ''
+    )
+
+    const handleClick = useCallback(
+      (e: MouseEvent<HTMLDivElement>) => {
+        onClick?.(e)
+        handleNavigate(e)
+      },
+      [handleNavigate, onClick]
+    )
+
+    if (!collection) return null
+
+    const {
+      playlist_name,
+      permalink,
+      playlist_owner_id,
+      repost_count,
+      save_count,
+      is_private,
+      access,
+      stream_conditions
+    } = collection
+
+    const isPurchase = isContentUSDCPurchaseGated(stream_conditions)
+
+    const dogEarType = is_private
+      ? DogEarType.HIDDEN
+      : isPurchase && !access.stream
+      ? DogEarType.USDC_PURCHASE
+      : null
+
+    return (
+      <Card ref={ref} onClick={handleClick} size={size} {...other}>
+        {dogEarType ? (
+          <DogEar type={dogEarType} css={{ top: -1, left: -1 }} />
+        ) : null}
+        <Flex direction='column' p='s' gap='s'>
+          <CollectionImage
+            collectionId={id}
+            size={cardSizeToCoverArtSizeMap[size]}
+            data-testid={`${id}-cover-art`}
+          />
+          <Flex direction='column' gap='xs'>
+            <Text variant='title' color='default' ellipses asChild>
+              <Link to={permalink} css={{ pointerEvents: 'none' }}>
+                {playlist_name}
+              </Link>
+            </Text>
+            <UserLink
+              userId={playlist_owner_id}
+              textVariant='body'
+              css={{ justifyContent: 'center' }}
+            />
+          </Flex>
+        </Flex>
+        <CardFooter>
+          {is_private ? (
+            <Text variant='body' size='s' strength='strong' color='subdued'>
+              {messages.hidden}
+            </Text>
+          ) : (
+            <>
+              <Flex gap='xs' alignItems='center'>
+                <IconRepost size='s' color='subdued' title={messages.repost} />
+                <Text variant='label' color='subdued'>
+                  {repost_count}
+                </Text>
+              </Flex>
+              <Flex gap='xs' alignItems='center'>
+                <IconHeart
+                  size='s'
+                  color='subdued'
+                  title={messages.favorites}
+                />
+                <Text variant='label' color='subdued'>
+                  {save_count}
+                </Text>
+              </Flex>
+              {isPurchase ? (
```

Comment:
are we meant to show the pill even if it's locked for other reasons?

Reply:
for collections i think this is the only case right?